Makefile

```diff
@@ -147,7 +147,7 @@ $(build_depsbindir)/stringreplace: $(JULIAHOME)/contrib/stringreplace.c | $(buil
 	@$(call PRINT_CC, $(HOSTCC) -o $(build_depsbindir)/stringreplace $(JULIAHOME)/contrib/stringreplace.c)
 
 julia-base-cache: julia-sysimg-$(JULIA_BUILD_MODE) | $(DIRS) $(build_datarootdir)/julia
-	@$(call spawn,$(JULIA_EXECUTABLE) --startup-file=no $(call cygpath_w,$(JULIAHOME)/etc/write_base_cache.jl) \
+	@$(call spawn,$JULIA_BINDIR=$(build_bindir) (JULIA_EXECUTABLE) --startup-file=no $(call cygpath_w,$(JULIAHOME)/etc/write_base_cache.jl) \
```

Comment:
I think this `$JULIA_BINDIR` is a mistake; I'm pretty sure that will expand to `ULIA_BINDIR`.

Reply:
oops, thanks

Reply:
This also broke building with wine:

```
cd /mnt/julia/base && if ! wine JULIA_BINDIR=`winepath -w /mnt/build/usr/bin` /mnt/build/usr/bin/julia.exe -g1 -O0 -C "pentium4" --output-ji `winepath -w /mnt/build/usr/lib/julia/sys.ji`.tmp  --startup-file=no --warn-overwrite=yes --sysimage `winepath -w /mnt/build/usr/lib/julia/corecompiler.ji` sysimg.jl ../../build/base/; then echo '*** This error might be fixed by running `make clean`. If the error persists, try `make cleanall`. ***'; false; fi
wine: cannot find 'JULIA_BINDIR=Z:\mnt\build\usr\bin'
```